src/request/node.ts

```diff
@@ -1,334 +0,0 @@
-import Task from '../async/Task';
```

Comment:
So.. I played around with a few ways to get git to recognize this as a rename rather than a new file. I think that it worked.. but it dawned on me that since we only squash from github, that special 'renaming commit' will get squashed into the other commits and we'd lose that history anyways, ending up with a single "'delete xyz', 'add xyy'" flow with files so different that git won't think they are a rename (the problem we have now).